scanner/src/test/kotlin/scanners/fossid/FossIdTest.kt

```diff
@@ -103,6 +103,18 @@ class FossIdTest : WordSpec({
         }
     }
 
+    "convertGitUrlToProjectName()" should {
+        "extract the repository name from the git URL without the .git suffix" {
+            val httpGitUrl = FossId.convertGitUrlToProjectName("http://github.com/path/to/example-repo.git")
```

Comment:
The variable names, e.g. `httpGitUrl` do not hold and URL but the project name as value.
How about inlining the variables? Maybe the assertions fit on one line if you add a static import for the convert function.